pages/1.11/security/ent/secrets/create-secrets/index.md

```diff
@@ -77,7 +79,25 @@ This procedure describes how to create a key/value pair secret called `my-secret
    dcos security secrets create --value=top-secret developer/my-secret
    ```
 
-# Creating secrets from a file via the DC/OS Enterprise CLI
+# Creating secrets from a any file via the DC/OS Enterprise CLI
+
+This procedure describes how to use any file to create a secret called `myapp.keystore` inside the `developer` path using the DC/OS Enterprise CLI.
+
+The contents of the file (referred to below as `myapp.keystore`) can be any value, including binary and text value. The secrets service encodes binary files into `BASE64` text form before storing it in secret store.
+
+1. Use `dcos auth login` to log into the CLI.
+
+1. Use the following command to create the new secret.
+
+  ```bash
+  dcos security secrets create -f myapp.keystore developer/keystore
+  ```
+
+   **Important:** The maximum file size for a `BASE64` encoded file is approximately one MiB, subtracting approximately one KB for the secret store metadata.
```

Comment:
Is it 1 MiB before or after encoding? I think what I want to know as a user is (approximately) how big my file can be before sending it.

Reply:
If this cannot be known, maybe give instructions to b64encode and see the file size locally?

Reply:
Answer is here: https://stackoverflow.com/a/4715480/145400
That is, the limit should be around 768 KiB.